### PR TITLE
Extend GHA autolabeler to other areas

### PR DESCRIPTION
**Pull request by abhishekrb19**
Extending the docs label added by @317brian, this PR adds labels to PRs broadly to other areas using GHA labeler. The labels are adopted from [here]( https://github.com/apache/druid/blob/master/dev/committer-instructions.md). The list of automated labels added in this PR is not complete yet.

This PR has:

- [x] been self-reviewed.

### CODE REVIEW

**Review by tejaswini-imply — COMMENTED** [review]
LGTM syntax-wise!

File: .github/labeler.yml [hunk 1]
```diff
@@ -18,9 +18,44 @@
...
 
-"Area - Documentation":
-  - "docs/**/*"
-  - "website/**"
-  - "examples/quickstart/jupyter-notebooks/**"
\ No newline at end of file
+'Area - Batch Ingestion':
+  - 'indexing-hadoop/**'
+  - 'extensions-core/multi-stage-query/**'
+
```
[the remark this answers is not in the document]
Comment (abhishekrb19) [reply .github/labeler.yml:26]: Yeah, I think more generally ingestion? I've added a label for that

**Review by tejaswini-imply — APPROVED** [review]

**Comment by abhishekrb19** [comment]
Thanks for the review, @tejaswini-imply. Tested the changes [here](https://github.com/abhishekrb19/incubator-druid/pull/11).

File: .github/labeler.yml [hunk 2]
```diff
@@ -18,9 +18,50 @@
...
+  - '**/pom.xml'
+  - 'licenses.yaml'
+
+'Area - Documentation':
+  - 'docs/**/*'
+  - 'website/**'
+  - 'examples/quickstart/jupyter-notebooks/**'
+
+'Area - Ingestion':
+  - 'processing/**'
```
Comment (clintropolis) [inline .github/labeler.yml:37]: this isn't really necessarily true, processing could also be querying

**Pull request merged**